sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -228,6 +228,16 @@ func (ct containerType) Prefix() string {
 	return ""
 }
 
+// SliceContains returns true if a slice contains a string
+func SliceContains(s []string, v string) bool {
```

Comment:
nit: Do we anticipate exporting `SliceContains` out of this package (i.e., will other Go programs import this and use it)? If not, this should be `sliceContains` - it probably makes the most sense to keep it private to this package